app/Controllers/ApiController.php

```diff
@@ -40,12 +40,9 @@ public function metadata()
                 'updated_at'           => $this->request->request->get('updated_at'),
                 'total_pages'          => $this->request->request->get('total_pages'),
                 'supporting_contracts' => $this->request->request->get('supporting_contracts'),
+                'published_at'         => $this->request->request->get('published_at'),
             ];
 
-            if (!empty($this->request->request->get('published_at'))) {
```

Comment:
check if date is valid instead of empty check

Reply:
Done